runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -560,10 +559,10 @@ private boolean checkAndModify(MenuEntry old)
 			AgilityShortcut shortcut = nearbyObstacle.getShortcut();
 			if (shortcut != null && Ints.contains(shortcut.getObstacleIds(), old.getIdentifier()))
 			{
-				final int reqLevel = shortcut.getLevel();
-				final String requirementText = ColorUtil.getLevelColorString(reqLevel, getAgilityLevel()) + "  (level-" + reqLevel + ")";
+				//final int reqLevel = shortcut.getLevel();
+				//final String requirementText = ColorUtil.getLevelColorString(reqLevel, getAgilityLevel()) + "  (level-" + reqLevel + ")";
```

Comment:
wasn't sure what to do here so i ended up changing "requirementText" to "shortcut.requirements"